src/background.ts

```diff
@@ -96,7 +96,7 @@ browser.runtime.onStartup.addListener(_ => {
         } else {
             native.run("hostname").then(hostname => {
                 for (const host of hosts) {
-                    if ((new RegExp(host)).exec(hostname.content)) {
+                    if (new RegExp(host).exec(hostname.content)) {
```

Comment:
Does `new` really bind this tightly? I think this is a bad prettification rule as it is very un-obvious to me.